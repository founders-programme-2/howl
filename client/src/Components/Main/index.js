import React from 'react';

const Main = () => {
  return (
    <main>
      <h1>Hello Main page!</h1>
    </main>
  );
};

export default Main;
