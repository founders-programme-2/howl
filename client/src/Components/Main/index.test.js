import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import Main from './index';

test('Natural', () => {
  const main = renderer
    .create(
      <BrowserRouter>
        {' '}
        <Main />{' '}
      </BrowserRouter>
    )
    .toJSON();
  expect(main).toMatchSnapshot();
});
