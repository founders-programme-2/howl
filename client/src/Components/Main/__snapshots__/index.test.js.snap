// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Natural 1`] = `
Array [
  " ",
  <main>
    <h1>
      Hello Main page!
    </h1>
  </main>,
  " ",
]
`;
